app/views/devise/passwords/edit.html.haml

```diff
@@ -8,5 +8,5 @@
       = f.input :password, label: "New password", required: true, autofocus: true, hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length)
       = f.input :password_confirmation, label: "Confirm your new password", required: true
     .form-actions
-      = f.button :submit, "Change my password"
+      = f.button :button, "Change my password"
```

Comment:
It makes a `<button>` rather than a `<submit>`. You need either that or `role="button"` for the ui-kit styles to apply properly and I have a vague preference for `<button>`.